packages/MarkdownEditor-Tests.package/MarkdownTextStylerTest.class/instance/testApplyingThemeDoesNotThrowError.st

```diff
@@ -0,0 +1,9 @@
+as yet unclassified
+testApplyingThemeDoesNotThrowError
+
+	| textMorph |
+	textMorph := PluggableTextMorphPlus new
```

Comment:
Maybe this could go into `MarkdownEditorTests`? The `applyUserInterfaceTheme` method does not exist in Squeak 5.0. `MarkdownEditor open` would then be unnecessary.

```suggestion
	MarkdownEditor open.
        self shouldnt: [CommunityTheme new apply] raise: Error
```

Reply:
Ah no worries, I didn't see Squeak 5.0 doesn't even have themes... 😞